specs/experimental/gov-delegation.md

```diff
@@ -309,17 +305,28 @@ enum AllowanceType {
 | `Absolute`  | `0`    | The amount of votes delegated is fixed and denominated in the `GovernanceToken`'s decimals  |
 | `Relative`  | `1`    | The amount of votes delegated is relative and denominated in percentages.                   |
 
-## Backwards Compatibility
+## Migration
 
-The `GovernanceDelegation` contract ensures backwards compatibility by allowing the migration of delegation state from the
-`GovernanceToken`.
+All write functions in the `GovernanceDelegation` MUST check if the users interacting with it (such as creating or updating
```

Comment:
Technically we don't need this in *all* write functions, only in functions that would create a delegation